www/style/codecheckerviewer.css

```diff
@@ -38,7 +38,14 @@ html, body {
   text-align: center;
 }
 
-.mainMenuButton {
+#loggedin {
+  padding-right: 10px;
+}
+
+#header-menu {
+  font-family: 'Source Code Pro', monospace;
```

Comment:
I've been tweaking a bit with the appearance of this text here. I would like to suggest a better idea.

First of all, the font specifications should go into `#loggedin`'s selector, not `#header-menu`'s.

And the settings I think that look much nicer than a monospace font:

```css
  font-family: 'Raleway', sans-serif;
  font-size: 11pt;
  color: white;
```

Compare:

![image](https://user-images.githubusercontent.com/1969470/28560740-24fbbad6-711c-11e7-8be6-5c55c664bc17.png)

to

![image](https://user-images.githubusercontent.com/1969470/28560712-0b007248-711c-11e7-8d6d-0f3364233eda.png)